apps/iglidur-designer/src/app/libs/header/src/lib/header/header.component.ts

```diff
@@ -53,6 +55,7 @@ export class HeaderComponent implements OnInit, OnDestroy {
       .subscribe((language) => {
         this.language = language;
         this.translate.use('header.component.i18n');
+        this.changeDetectorRef.markForCheck();
```

Comment:
Are you sure this is necessary?

Reply:
It still works without me adding it. But I read a resource that recommended adding it.
I'll remove it then.

Reply:
Yes you need it if you really need to notify angular that there is a change that should be reflected in the view  but in this case you are subscribing on initialization and keeping the stream open which make it reactive :)

Reply:
Great.. thanks for the explanation